### ISSUE TITLE
Transparency is not working

### ISSUE THREAD

**Issue by Shell1010** [QNetITQ/WaveFox#193]
Using Manjaro KDE with Force Blur and Firefox 123.

Installed WaveFox v1.6.123

![image](https://github.com/QNetITQ/WaveFox/assets/72198971/2c639808-0e2b-425f-8559-73517c9de1ab)
![image](https://github.com/QNetITQ/WaveFox/assets/72198971/8abf37df-af20-4bae-9542-41dbd4a5c12a)
![image](https://github.com/QNetITQ/WaveFox/assets/72198971/e9d884e9-4049-45cb-88d7-66abc195dab1)

The different tab options work, but I haven't been able to get transparency working. This works for my other applications, just seeming to not work here.

**Comment by Shell1010** [comment]
Does this need to be in light theme for it to work?

**Comment by QNetITQ** [comment]
You can choose any theme, be it light or dark. I am working. Maybe you didn't add the Firefox process to Force Blur or did it incorrectly.
![изображение](https://github.com/QNetITQ/WaveFox/assets/85301851/6e918267-37a2-4331-bdb8-ba99a3b069eb)

**Comment by Shell1010** [comment]
Yes I've also got forceblur set like this, works for other applications too.

![image](https://github.com/QNetITQ/WaveFox/assets/72198971/64d888a7-652a-405f-aca9-ecd74fbcbe72)

I activated the following keys step by step.
`toolkit.legacyUserProfileCustomizations.stylesheets`
`svg.context-properties.content.enabled`
`userChrome.Tabs.Option5.Enabled`
`userChrome.Linux.Transparency.High.Enabled`
`gfx.webrender.all`
`browser.tabs.inTitlebar` set to 2.

Anything I've missed?
